### PR TITLE
feat(rust): add hello world example

### PR DESCRIPTION
**Pull request by georgettica**
related to #28

### CODE REVIEW

**Review by rustiever — CHANGES_REQUESTED** [review]
please update language in readme file,
typo in line number 10,

**Comment by georgettica** [comment]
@rustiever hope I fixed the README as you wanted, waiting for your approval

**Review by rustiever — APPROVED** [review]
LGTM

**Pull request merged**
